.github/workflows/pipeline.yml

```
@@ -0,0 +1,195 @@
+        pip install . --no-deps --no-build-isolation
+        python -m unittest discover tests
+
+  autobot:
+    needs: [unittest_old, unittest_matrix, pip_check, coverage]
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    if: (github.event.pull_request.user.login == 'dependabot[bot]' || github.event.pull_request.user.login == 'pre-commit-ci[bot]')
+    steps:
+      - name: Enable auto-merge for bot PRs
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)